src/index.js

```diff
@@ -31,11 +34,16 @@ async function runTest(files, options) {
         let returnVals = await brs.execute(allTestFiles, {
             root: process.cwd(),
             stdout: reporterStream,
-            stderr: process.stderr
+            stderr: process.stderr,
+            generateCoverage: coverageEnabled
         });
 
         reporterStream.end();
 
+        if (coverageEnabled) {
+            coverage.report(coverageReporters);
```

Comment:
This might cause issues for users who provide `--reporter tap`, `--reporter json`, or `--reporter json-stream`, since it causes `stdout` to contain both TAP (and TAP-in-JSON) and coverage output. I don't mind that behavior for `roca` — especially for our initial pass — but I'm curious how something like `mocha` handles that!

The biggest practical impact is that running `roca --coverage-reporters=html > coverage-report.html` won't produce a valid HTML file.

Reply:
Hmmm interesting. A couple of thoughts: if you didn't want TAP output to pollute coverage output, you could do `--reporter silent` (i agree that's an interim solution, not long term). But also, running `--coverage-reporters=html` actually generates a `coverage/` folder, and the coverage report is written to `coverage/index.html`. So I don't know that there's necessarily a use case where you would need to pipe stdout to an html file! Of course, you may want to pipe other output formats, such as `--coverage-reporters=text`, which would still be an issue.

But yeah, i think this should work for a first pass.

Reply:
> But also, running --coverage-reporters=html actually generates a coverage/ folder in the project root, and the coverage report is written to coverage/index.html.

Heh I actually didn't know it produced a separate `coverage/` directory.  `--coverage-reporters=text` is exactly the use-case I was thinking of 🙂